xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/pom.xml

```diff
@@ -64,13 +64,19 @@
     <dependency>
       <groupId>io.github.bonigarcia</groupId>
       <artifactId>webdrivermanager</artifactId>
-      <version>5.1.1</version>
+      <version>5.3.1</version>
```

Comment:
was this needed or is it something separate?

Reply:
This dependency is used in the `com.deque.html.axe-core` package too, and leaving the version to 5.1.1 fails the build.

The build logs without this version change:

>  [INFO] --- maven-enforcer-plugin:1.4.1:enforce (enforce-upper-bounds) @ xwiki-platform-test-ui ---
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for io.github.bonigarcia:webdrivermanager:5.1.1 paths to dependency are:
+-org.xwiki.platform:xwiki-platform-test-ui:15.1-SNAPSHOT
  +-io.github.bonigarcia:webdrivermanager:5.1.1
and
+-org.xwiki.platform:xwiki-platform-test-ui:15.1-SNAPSHOT
  +-com.deque.html.axe-core:selenium:4.6.0
    +-io.github.bonigarcia:webdrivermanager:5.3.1
]